pkg/runtime/wfengine/component.go

```diff
@@ -43,6 +43,9 @@ var ComponentDefinition = componentsV1alpha1.Component{
 	},
 }
 
+// This error was obtained from backend.go from the durabletask.go package
+var ErrInstanceNotFound = errors.New("No such instance exists")
```

Comment:
Shouldn't you be using backend.ErrInstanceNotFound instead of defining your own version here?